app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -50,7 +50,7 @@
     private BluetoothAdapter bluetoothAdapter;
     private IBluetoothHandler bluetoothHandler;
     private Set<BluetoothDevice> pairedDevices;
-    private boolean isMaster = false;
+    private boolean isMaster;
```

Comment:
Technically speaking nothing, but if you set Intellij checking on the highest level, then it gives a warning that this clutters the codebase. For experienced developers it's clear that the value is initialized to false, so the assignment is redundant